Sources/MuxPlayerSwift/PublicAPI/Extensions/AVPlayerItem+Mux.swift

```diff
@@ -83,6 +83,24 @@ fileprivate func makePlaybackURL(
     return playbackURL
 }
 
+/// Create a new `AVAsset` that has been preparted for playback
+/// Currently, being "prepared for playback" means only that the DRM token was added to the asset's options if present
+fileprivate func makeAVAsset(playbackID: String, playbackOptions: PlaybackOptions) -> AVAsset {
+    let url = makePlaybackURL(playbackID: playbackID, playbackOptions: playbackOptions)
+    
+    var asset: AVURLAsset
+    if case .drm(let options) = playbackOptions.playbackPolicy {
+        PlayerSDK.shared.fairplaySessionManager.registerDrmToken(options.drmToken, for: playbackID)
+        asset = AVURLAsset(url: url)
+        // asset must be attached as early as possible to avoid crashes when attaching later
+        PlayerSDK.shared.fairplaySessionManager.contentKeySession.addContentKeyRecipient(asset)
```

Comment:
Was thinking about how to make this easier to test, at least on our side of the fence. 

What do you think about passing in the `fairplaySessionManager` as an additional argument? We'd then be able to (in the future) use a mock to validate that we make / skip FPS-related calls.